openapi/schemas/v2/Markup/GetRedlines.yml

```
@@ -0,0 +1,112 @@
+---
+openapi: 3.0.0
+info:
+  version: '2.0'
+  title: Item Attachment Put
+  license:
+    name: MIT
+servers:
+- url: https://{orgId}.salesforce.com/services/apexrest/{namespace}/v2
```

highly recommend endpoint > orgId. most people don't know their org id off the top of their head, whereas base endpoint is a bit more intuitive. orgId is also a misnomer as I assume you want org instance (e.g. NA159) not the actual org id, though I could be wrong.
```suggestion
- url: https://{endpoint}/services/apexrest/{namespace}/v2
```